stripe.asd

```diff
@@ -1,14 +1,17 @@
-(asdf:defsystem #:stripe
+(defsystem #:stripe
   :description "A client for the Stripe payment API."
   :author ("Michael Fiano <mail@mfiano.net>")
   :maintainer "Atlas Engineer LLC"
   :license "MIT"
   :homepage "https://github.com/atlas-engineer/stripe"
   :encoding :utf-8
-  :depends-on (#:dexador
+  :depends-on (#:alexandria
+               #:com.inuoe.jzon
+               #:dexador
                #:golden-utils
+               #:ironclad
                #:local-time
-               #:yason)
+               #:str)
```

Comment:
Do we really need to pull `str`? It's ok to pull `alexandria` since it's already a dependency of `dexador`.

`str:ends-with-p` can be probably be replaced by `alex:ends-with`. `str:replace-first` can probably be replaced by `substitute`.

Reply:
How would you use substitute here?

```
(str:replace-first "item" "-item" object-type)
```

Reply:
bump @aadcg

Reply:
I'm not sure I understand the question. Doesn't `str:replace-first` fail on the special you mention above?